ONSdigital/babbage
# Feature/release banner compendium

### What & Why
#### What
* Release banner now added to the compendiums landing page
* Compendium headers refactored into their own handle bars partial 
* Bulletin header refactored into their own handlebars partial
* Release Banner refactored and treated like a component (new handlebars partial)

 #### Why
The code had a bad smell. It did it follow the Open Closed Principle as such expansion of the release calendar without a refactor would require the code to then break DRY (don't repeat yourself) principles.

### How to review

* Checkout the corresponding Sixteens PR: https://github.com/ONSdigital/sixteens/pull/144
* Check that the header section of each type of page is as expected (un changed unless it is a compendium)
* Ensure that the compendium landing page shows the new release banner which can typically be found on bulletins
* Ensure that the code meets best practices.

It should look like the following:
<img width="833" alt="Screenshot 2019-04-04 at 14 59 20" src="https://user-images.githubusercontent.com/47502916/55561585-4f0b7e00-56ea-11e9-9a1c-143f488f50a3.png">
<img width="789" alt="Screenshot 2019-04-04 at 14 59 27" src="https://user-images.githubusercontent.com/47502916/55561588-516dd800-56ea-11e9-867e-d1f638ac6b9d.png">

Note: The Welsh translations have yet to be added, the box should also size to the required width depending on the length of the text. 

### Who can review

Anyone except me on the dev team
